src/Popover/components/ContentWrapper.js

```diff
@@ -23,38 +23,50 @@ import { StyledButtonPrimitive } from "../../primitives/ButtonPrimitive";
 
 const mobileTop = theme => theme.orbit.spaceXLarge;
 const popoverPadding = theme => theme.orbit.spaceMedium;
-const actionsSpace = theme => theme.orbit.spaceMedium;
-
-const allSpacing = theme =>
-  parseFloat(popoverPadding(theme)) * 2 +
-  parseFloat(mobileTop(theme)) +
-  parseFloat(actionsSpace(theme));
 
 const StyledContentWrapper = styled.div`
   overflow: auto;
-  max-height: ${({ actionsHeight, theme }) =>
+  max-height: ${({ actionsHeight, windowHeight }) =>
     // Calculates all the spacing relative to viewport to get space for action box
-    `calc(100vh - ${allSpacing(theme) + actionsHeight}px)`};
+    `${windowHeight - actionsHeight - 34}px`};
```

Comment:
What's `34 pixels`? 👀 Can you at least put a comment there? 🙏

Reply:
sure I can it is the same magic number as in modal